Detections/MultipleDataSources/MFADisable.yaml

```diff
@@ -0,0 +1,42 @@
+id: 65c78944-930b-4cae-bd79-c3664ae30ba7
+name: MFA disable for a user
+description: |
+  'Multi-Factor Authentication (MFA) helps prevent credential compromise. This alert identifies when an attempt has been made to diable MFA for a user '
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+     - AuditLogs
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1098
+query: |
+ let timeframe = 1d;
+ (union isfuzzy=true
+ (AuditLogs 
+ | where TimeGenerated >= ago(timeframe) 
+ | where OperationName =~ "Disable Strong Authentication"
+ | extend IPAddress = tostring(parse_json(tostring(InitiatedBy.user)).ipAddress) 
+ | extend InitiatedByAdmin = iff(isnotempty(tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName)), 
+  tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName), tostring(parse_json(tostring(InitiatedBy.app)).displayName))
+ | extend Targetprop =todynamic(TargetResources)
+ | extend TargetUser = tostring(Targetprop[0].userPrincipalName) 
+ | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by User = TargetUser, InitiatedByAdmin , Operation = OperationName , CorrelationId, IPAddress, Category, Source = SourceSystem , AADTenant_or_AWSInstance_Detail = AADTenantId, Type
```

Comment:
You have this value - AADTenant_or_AWSInstance_Detail, but I don't see this in the AWSCloudTrail project.  Seems like you were intending to with the name?